auth/google.js

```diff
@@ -13,13 +17,13 @@ passport.deserializeUser(function (id, fn) {
 });
 
 passport.use(new GoogleStrategy({
-    clientID: "321138157267-llg1g0qv2kpp5lrg43sablqeot01libh.apps.googleusercontent.com",
-    clientSecret: "g8uRU_yMjyQpGTYsdBuxY_y7",
-    callbackURL: "http://127.0.0.1:3000/auth/google/callback"
+    clientID: process.env.GOOGLE_CLIENT_ID,
+    clientSecret: process.env.GOOGLE_CLIENT_SECRET,
+    callbackURL: CALLBACK_URL
```

Comment:
Callback url is something that is set up by google, its not just something you can change locally